nowcasting_dataloader/utils/position_encoding.py

```diff
@@ -308,18 +308,19 @@ def normalize_geospatial_coordinates(
     geospatial_coordinates[1] = geospatial_coordinates[1] * 2 - 1
     # Now create a grid of the coordinates
     # Have to do it for each individual example in the batch, and zip together x and y for it
-    to_concat = []
+    solid_tensor = None
     for idx in range(len(geospatial_coordinates[0])):
         x = torch.from_numpy(geospatial_coordinates[0][idx])
```

Comment:
Are `x` and `y` in their "minimal data types"?  (e.g. int16)?  Using the smallest data types possible might help speed things up?